servers/Azure.Mcp.Server/tests/Azure.Mcp.Server.UnitTests/Infrastructure/CommandMetadataSyncTests.cs

```diff
@@ -52,45 +53,56 @@ public void AzCommandsMetadata_Should_Be_Synchronized()
             "This ensures that tool metadata in the documentation stays synchronized with the actual command implementations.");
     }
 
-    private static (int ExitCode, string Output, string Error) RunCommand(string command, string arguments, string workingDirectory)
+    private static string FindPowerShellExecutable()
     {
-        var processInfo = new ProcessStartInfo
-        {
-            FileName = command,
-            Arguments = arguments,
-            WorkingDirectory = workingDirectory,
-            RedirectStandardOutput = true,
-            RedirectStandardError = true,
-            UseShellExecute = false,
-            CreateNoWindow = true
-        };
-
-        using var process = new Process { StartInfo = processInfo };
-        var outputBuilder = new System.Text.StringBuilder();
-        var errorBuilder = new System.Text.StringBuilder();
-
-        process.OutputDataReceived += (sender, args) =>
+        // Try common PowerShell paths
+        var commonPaths = OperatingSystem.IsWindows()
```

Comment:
Why do we need to prob these paths? We should just try to resolve pwsh from the path so call it by name and if it isn't there handle the error condition and tell folks to install it.